components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -674,19 +680,16 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
             },
         );
 
-        if (this.user) {
-            await this.userToTeamMigrationService.migrateUser(this.user.id, true, "on demand");
-            let updatedUser = await this.userDB.findUserById(this.user.id);
-            if (updatedUser) {
-                this.user = updatedUser;
-            }
+        if (this.userID) {
+            await this.userToTeamMigrationService.migrateUser(this.userID, true, "on demand");
+            await this.userDB.findUserById(this.userID);
```

Comment:
side note: We don't need this call anymore.

Reply:
removed.